src/wordnet_autotranslate/gui/synset_browser.py

```
@@ -825,24 +1082,228 @@ def _extract_english_id(self, synset_id: str) -> Optional[str]:
+            
+        Returns:
+            Serbian WordNet ID format like 'ENG30-{offset}-{pos}'
+        """
+        if not english_synset_name or english_synset_name == 'N/A':
+            return 'N/A'
+        
+        try:
+            # Parse the English synset name (e.g., 'dog.n.01')
+            parts = english_synset_name.split('.')
+            if len(parts) >= 2:
+                word = parts[0]
```

Ruff (0.12.2)

Local variable `word` is assigned to but never used

Remove assignment to unused variable `word`

(F841)